### PR TITLE
Changed logic of AccidentCountByDriverTypeWidget

### PR DESCRIPTION
**Pull request by volsky**
Added filtering to include only drivers
Added count distinct

### CODE REVIEW

**Comment by atalyaalon** [comment]
@volsky looks good! please fix Black tests
You can use:
`python -m pip install --upgrade black`
`black anyway/**/*.py -l 100 --verbose --check`

File: anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py [hunk 1]
```diff
@@ -29,11 +29,14 @@ def generate_items(self) -> None:
 
     @staticmethod
     def count_accidents_by_driver_type(request_params):
+        filters = get_injured_filters(request_params.location_info)
+        filters["involved_type"] = [1,2]
```
Comment (atalyaalon) [inline anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py:33]: looks good, perhaps we should add it to our constants (backend_constants.py file) ?

**Review by atalyaalon — APPROVED** [review]
Approved, aside of black modifications

File: anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py (hunk 1 above)
Comment (volsky) [reply anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py]: done

**Comment by volsky** [comment]
> @volsky looks good! please fix Black tests You can use: `python -m pip install --upgrade black` `black anyway/**/*.py -l 100 --verbose --check`

Done, there were some other files (unrelated to this change) that changed because of black

**Pull request merged**